momento-sdk/src/main/java/momento/sdk/ScsDataClient.java

```diff
@@ -619,28 +620,29 @@ CompletableFuture<CacheSortedSetGetScoreResponse> sortedSetGetScore(
   }
 
   CompletableFuture<CacheSortedSetGetScoresResponse> sortedSetGetScores(
-      String cacheName, String sortedSetName, Set<String> elements) {
+      String cacheName, String sortedSetName, Iterable<String> values) {
     try {
       checkCacheNameValid(cacheName);
       checkSetNameValid(sortedSetName);
-      ensureValidValue(elements);
+      ensureValidValue(values);
```

Comment:
I always get tripped up on this. The `value` is the name of the element stored in the sorted set?

Reply:
Element refers to the value and score. An element can be indexed by its value or rank.